Demos/01-create-app/graph-tutorial/Content/Site.css

```diff
@@ -1,7 +1,7 @@
-﻿/*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT license.
- */
+﻿/* 
```

Comment:
There are a lot of files here that only have whitespace changes, or only have `using` statements in a different order. Let's undo those changes to slim down this PR please.

Reply:
Fixed... brought number of files from 43=>25